### PR TITLE
apply SL suffix to Layoutable

### PR DESCRIPTION
**Pull request by oozoofrog**
#156

### CODE REVIEW

File: Sources/SwiftLayout/Layoutable/Wrapper/SL.swift [hunk 1]
```diff
@@ -0,0 +1,28 @@
+//
+//  SL.swift
+//  
+//
+//  Created by oozoofrog on 2022/03/13.
+//
+
+import Foundation
+
+public final class SL<L: Layoutable> {
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Layoutable/Wrapper/SL.swift:10]: In fact, the name "SL" is ambiguous class name.
It's a bit long, but I think you need a clear name like "LayoutableMethodWrapper"

File: Sources/SwiftLayout/Layoutable/Wrapper/SL.swift [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    private weak var layoutable: L?
+    
+    init(_ layoutable: L) {
+        self.layoutable = layoutable
+    }
+    
+    public func layout() {
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Layoutable/Wrapper/SL.swift:17]: In fact, also the name "layout" is ambiguous.
Even if sl is attached, a clear name like "updateLayout" would be better.

File: Sources/SwiftLayout/Layoutable/Wrapper/SL.swift [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        guard let layoutable = layoutable else {
+            return
+        }
+        layoutable.activation = Activator.update(layout: layoutable.layout,
+                                                 fromActivation: layoutable.activation ?? Activation())
+    }
+}
+
+extension Layoutable {
+    public var sl: SL<Self> { .init(self) }
```
Comment (gmlwhdtjd) [inline Sources/SwiftLayout/Layoutable/Wrapper/SL.swift:27]: 👍

**Review by gmlwhdtjd — COMMENTED** [review]
Thanks for the good suggestion and implementation.
However, the naming seems to need some changes.

File: Sources/SwiftLayout/Layoutable/Wrapper/SL.swift (hunk 2 above)
Comment (oozoofrog) [reply Sources/SwiftLayout/Layoutable/Wrapper/SL.swift:17]: agreed

**Comment by oozoofrog** [comment]
> Thanks for the good suggestion and implementation.
> However, the naming seems to need some changes.

very agreed

File: Sources/SwiftLayout/Layoutable/Wrapper/LayoutableMethodWrapper.swift [hunk 4]
```diff
@@ -14,7 +14,7 @@ public final class LayoutableMethodWrapper<L: Layoutable> {
         self.layoutable = layoutable
     }
     
-    public func layout() {
+    public func updateLayout() {
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Layoutable/Wrapper/LayoutableMethodWrapper.swift:17]: renamed @gmlwhdtjd

File: Sources/SwiftLayout/Layoutable/Wrapper/LayoutableMethodWrapper.swift [hunk 5]
```diff
@@ -1,13 +1,13 @@
 //
-//  SL.swift
+//  LayoutableMethodWrapper.swift
```
Comment (oozoofrog) [inline Sources/SwiftLayout/Layoutable/Wrapper/LayoutableMethodWrapper.swift:2]: renamed @gmlwhdtjd

**Comment by gmlwhdtjd** [comment]
👍

**Pull request merged**